src/init.cpp

```diff
@@ -810,6 +810,11 @@ void ThreadImport(std::vector<boost::filesystem::path> vImportFiles)
     if (activeMasternodeManager && fDIP0003ActiveAtTip)
         activeMasternodeManager->Init();
 
+#ifdef ENABLE_WALLET
+    // we can't do this before DIP3 is fully initialized
+    pwalletMain->AutoLockMasternodeCollaterals();
```

Comment:
should probably also wrap this in `if (pwalletMain)`

Reply:
```suggestion
    if (pwalletMain) {
        pwalletMain->AutoLockMasternodeCollaterals();
    }
```